### PR TITLE
Bug/d3 asim 1862

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: integration_tests/steps/jira_issues_tests.py [hunk 1]
```diff
@@ -284,3 +284,16 @@ def trades_happen(context):
 @given('the file {setup_json} is used for the area setup')
 def json_setup_file(context, setup_json):
     os.environ["D3A_SETUP_PATH"] = os.path.join(d3a_path, 'setup', setup_json)
+
+
+@then('the DSO should only pay 30 cents')
+def dso_pays_certain_price(context):
```
Comment (spyrostz) [inline integration_tests/steps/jira_issues_tests.py]: I would also add an initial check in this function, that validates that there is at least one trade in any market. That way it is going to be checked that at least one trade has been performed and the test will be executed correctly.

Comment (hannesdiedrich) [reply integration_tests/steps/jira_issues_tests.py]: Ok, will do!

Comment (faizan2590) [reply integration_tests/steps/jira_issues_tests.py]: May be pass rate as a parameter. Not necessary now but would be handy later for using the same method for grid fee testing.

Comment (hannesdiedrich) [reply integration_tests/steps/jira_issues_tests.py]: Hm, this test is really special. To make it more general, more stuff has to be added (to also take other grid structures). Do you still want me to change it?

Comment (faizan2590) [reply integration_tests/steps/jira_issues_tests.py]: Hmm, okay leave it as it is for now. If needed, could be modified later too.

**Pull request merged**